init.el

```diff
@@ -102,110 +276,264 @@
   (erc-mode . (lambda ()
                 (nyan-mode -1)
                 (setq show-trailing-whitespace nil))))
-
 (use-package erc-hl-nicks
-    :after erc)
-
+  :after erc)
 (use-package erc-image
-    :after erc)
+  :after erc)
+(use-package erc-yt)
+(use-package erc-colorize)
+
+;;; Navigation
+(use-package evil
+  :defer nil
+  :config
+  (setq evil-insert-state-map (make-sparse-keymap))
```

Comment:
Can you add

```suggestion
  (setq evil-insert-state-map (make-sparse-keymap))
  (setq evil-disable-insert-state-bindings t)
```

This should allow emacs keys to be enabled in vi-insert mode. odd how this happens in my normal config